src/main/resources/jenkins/plugins/slack/SlackNotifier/global.jelly

```diff
@@ -25,6 +25,9 @@
     <f:entry title="Is Bot User?" help="/plugin/slack/help-globalConfig-botUser.html">
         <f:checkbox field="botUser" />
     </f:entry>
+    <f:entry title="Send as Text?" help="/plugin/slack/help-globalConfig-sendAsText.html">
+        <f:checkbox field="sendAsText" />
```

Comment:
this isn't working currently

Reply:
Are you sure?  It's working fine for me.

Reply:
ah my bad it is working